File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
  int height;

  do
  {
      height = get_int("Introduce height: ");
  } while (height > 8 || height < 1);

  for(int i = 0; i < height; i++)
  {
    for(int j = 0; j <= i; j++)
    {
        printf("#");
    }
    printf("\n");
  }

}

File: Week6/pset6/mario/less/mario.py
from cs50 import get_int


def main():
    height = get_height()
    draw(height, height)


def get_height():
    while True:
        height = get_int("Height: ")
        if height>0 and height<9:
            break
    return height


def draw(height, h):
    if height == 0:
        return

    draw(height - 1, h)
    print(" " * (h - height), end='')
    print("#" * height)

main()
